packages/flutter_tools/lib/src/base/error_handling_file_system.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:io' as io show Directory, File, Link;
 
 import 'package:file/file.dart';
 import 'package:meta/meta.dart';
+import 'package:path/path.dart' as p; // ignore: package_path_import
```

Comment:
Why is importing package:path an analyzer warning?

Reply:
We want people to use our abstraction over it